### PR TITLE
ignore/skip urls with a fragment or query

### PR DESCRIPTION
**Pull request by romainmenke**
fixes : https://github.com/postcss/postcss-import/issues/537

### CODE REVIEW

File: lib/parse-styles.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ async function parseStyles(
     stmt.layer = joinLayer(layer, stmt.layer || [])
 
     // skip protocol base uri (protocol://url) or protocol-relative
```
Comment (RyanZim) [inline lib/parse-styles.js:30]:
```suggestion
```

File: lib/parse-styles.js [hunk 2]
```diff
@@ -219,4 +219,25 @@ async function loadImportContent(
   )
 }
 
+function isProcessableURL(uri) {
+  if (/^(?:[a-z]+:)?\/\//i.test(uri)) {
```
Comment (RyanZim) [inline lib/parse-styles.js:223]:
```suggestion
  // skip protocol base uri (protocol://url) or protocol-relative
  if (/^(?:[a-z]+:)?\/\//i.test(uri)) {
```

File: lib/parse-styles.js [hunk 3]
```diff
@@ -219,4 +219,25 @@ async function loadImportContent(
...
+    return false
+  }
+
+  try {
```
Comment (RyanZim) [inline lib/parse-styles.js:227]:
```suggestion
  // check for fragment or query 
  try {
```

**Review by RyanZim — APPROVED** [review]
Suggested a few comment changes

**Comment by romainmenke** [comment]
Thank you for the feedback and suggestions!

**Pull request merged**